person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rental
   attr_accessor :name, :age
 
   def initialize(age = nil, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] As required for the project, kindly ensure that a method for adding `rentals` is also present in the `Book` class. And for achieving the 'has-many' association, the person itself should be sent as a parameter to create the new rental. i.e.
```ruby
 def add_rental(book, date)
    Rental.new(date, book, self)
  end
```